src/plugins/hetero/tests/functional/CMakeLists.txt

```diff
@@ -3,14 +3,14 @@
 #
 
 set(TARGET_NAME ov_hetero_func_tests)
-
 ov_add_test_target(
         NAME ${TARGET_NAME}
         ROOT ${CMAKE_CURRENT_SOURCE_DIR}
         DEPENDENCIES
             mock_engine
             openvino_hetero_plugin
         LINK_LIBRARIES
+            openvino::funcSharedTests
```

Comment:
is it possible to avoid use of extra libraries? Looks like Hetero and Auto shared parts may become private